### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by frenzymadness** [ipython/ipyparallel#575]
Hello. Is there any plan for the next release 7.0.0? Jupyter-client 7.0.2 and ipykernel 6.3.1 are released and there are some commits for compatibility with them in the main branch.

I'd like to update ipykernel RPM package in Fedora and I don't know whether it is worth it to backport the compatibility patches or wait for the next release.

**Comment by minrk** [comment]
Yes, I'm working on it. Hopefully this month.

I wouldn't bother with the backports, I think it'll be hairy. If you can test the 7.0 prereleases, that will help me get the release ready.

**Comment by frenzymadness** [comment]
I gladly help you with testing and I'm gonna open separate issues if I find any bugs. In Fedora, we have bluepyopt and metakernel depending on ipyparallel so their tests should provide some value as well.

**Comment by frenzymadness** [comment]
I've prepared the RPM package for 7.0.0b3 and build it with all dependant packages after that. Both bluepyopt and metakernel built fine and their tests pass:
metakernel:
```
77 passed, 3 skipped, 1 deselected, 5 warnings in 36.45s
```
bluepyopt:
```
8 passed, 99 deselected in 124.64s (0:02:04)
```
From that point of view, everything looks good and ipyparallel is ready to be released. For more info see: https://copr.fedorainfracloud.org/coprs/lbalhar/ipyparallel/builds/

**Comment by minrk** [comment]
just published 7.0.0
